app/src/main/res/layout/fragment_launch_screen.xml

```diff
@@ -13,15 +13,14 @@
         style="@style/Text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/size_5x"
-        android:layout_marginBottom="@dimen/size_3x"
+        android:layout_marginTop="@dimen/size"
+        android:layout_marginEnd="@dimen/size_3x"
         android:fontFamily="@font/poppins_regular"
         android:letterSpacing="0.04"
         android:text="@string/launch_screen_txt_sign_in"
         android:textColor="@color/s_60"
         android:textSize="14sp"
```

Comment:
Don't we have a style for the text already? You shouldn't need to put fontFamily, letterSpacing etc.